README.md

```
@@ -1,209 +1,204 @@
+#### Fedora
 ```sh
 sudo dnf install cmake clang-devel libxkbcommon-devel systemd-devel expat-devel pcre2-devel libzstd-devel gtk3-devel
 make
 sudo make install
 ```
 
-**openSUSE:**
-
-Works with KDE Plasma (without GTK packages)
-
+#### openSUSE
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)